crates/consensus/consensus/Cargo.toml

```diff
@@ -16,9 +16,10 @@ reth-primitives.workspace = true
 
 # misc
 auto_impl.workspace = true
-thiserror-no-std = {workspace = true, default-features = false }
+thiserror-no-std = {workspace = true, default-features = false, optional = true }
```

Comment:
do we even still need this @shekhirin

Reply:
i don't think so, we manually implement the error now

Reply:
Removed